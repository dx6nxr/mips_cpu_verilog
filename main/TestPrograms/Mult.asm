ori $2, 100
ori $3, 238
multu $2, $3
mfhi $2
mflo $4