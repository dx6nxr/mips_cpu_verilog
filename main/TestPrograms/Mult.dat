34420064
346300ee
00430019
00001010
00002012