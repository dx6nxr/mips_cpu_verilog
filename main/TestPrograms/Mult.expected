@2 00000000
@3 000000ee
@4 00005cf8